### ISSUE TITLE
colorator gem issue

### ISSUE THREAD

**Issue by tomjoht** [bitmand/utterson#29]
When I try to install it, I get this error: 

```
Could not find colorator-0.1 in any of the sources
Run `bundle install` to install missing gems.
```

**Comment by bitmand** [comment]
@tomjohnson1492 that is wierd, colorator is actually a dependency for Jekyll itself and seems to exist just fine: https://rubygems.org/gems/colorator/versions/0.1

Just to be sure, you get that error when running `bundle install` ?

**Comment by tomjoht** [comment]
No, I get the error when running `bundle exec rackup`. The `bundle install` works fine. When I run `bundle exec rackup` I get:

```
Could not find colorator-0.1 in any of the sources
Run `bundle install` to install missing gems.
```

**Comment by bitmand** [comment]
I have tried updating the gems, just as a first try. Can you try:

```
git pull
bundle install
bundle exec rackup
```

Same error?
